### PR TITLE
Implementing reference speed change

### PR DESCRIPTION
**Pull request by dorinabajkai**
Review my solution Áron!

### CODE REVIEW

**Review by aronszabo — APPROVED** [review]
I like your coding style. I suggest merging to this code to the project.

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 1]
```diff
@@ -51,4 +51,16 @@ public void setJoystickPosition(int joystickPosition) {
 	public Tachograph getTg(){
 		return tg;
 	}
+
+	@Override
+	public void run() {
+		while(true){
+			followSpeed();
+			try {
+				Thread.sleep(1000);
```
Comment (aronszabo) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: However, you should move the train at a faster rate.

**Review by aronszabo — CHANGES_REQUESTED** [review]
Please change your code.

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java (hunk 1 above)
Comment (dorinabajkai) [reply train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: I'll lower it a bit

**Review by aronszabo — APPROVED** [review]
Yes, that is better.

**Pull request merged**